locales/en-US/app.ftl

```diff
@@ -102,6 +103,84 @@ ListOfPublishedProfiles--uploaded-profile-information-list =
        *[other] Manage these recordings
     }
 
+## MenuButtons Folder
```

Comment:
@canova I didn't ask about this. So the components inside `MenuButtons` are grouped together here. I was not sure if should add these as other components. Let me know what you think.